astrolabe/cli.py

```diff
@@ -174,16 +174,16 @@ def atlas_projects():
     pass
 
 
-@atlas_projects.command('create')
+@atlas_projects.command('ensure')
```

Comment:
This was changed because more often than not, this is what the user wants to do.